infoschema/builder.go

```diff
@@ -219,6 +219,8 @@ func (b *Builder) ApplyDiff(m *meta.Meta, diff *model.SchemaDiff) ([]int64, erro
 		return b.applyRecoverTable(m, diff)
 	case model.ActionCreateTables:
 		return b.applyCreateTables(m, diff)
+	case model.ActionFlashbackCluster:
+		return []int64{-1}, nil
```

Comment:
1. use `-1` to represent all tables and it easy expand to `flashback table` and `flashback database`
2. If use empty slice, it will not add action to `relatedChange.ActionTypes`. see codes:
https://github.com/pingcap/tidb/blob/6c55faf034e8c205ffd23126829c637fb8a47451/domain/domain.go#L337-L347